apps/communications/contacts/test/marionette/lib/contacts.js

```diff
@@ -410,6 +411,13 @@ Contacts.prototype = {
     return this.client.executeScript(function(selector, type) {
       return document.querySelector(selector).style[type];
     }, [selector, type]);
+  },
+
+  getNumberOfContacts: function() {
+    return this.client.executeScript(function() {
```

Comment:
This can be done thanks to Marionette's API.

```js
get contacts() {
  try {
     var contactElements = this.client.findElements(Selectors.contactItems);
     return contactElements.map(function(contactElement) { return contactElement.text() });
  } catch (e) {
     // No elements were found
     return [];  
  }
}
```